public/app/features/admin/UserListAdminPage.tsx

```diff
@@ -149,6 +179,16 @@ const renderUser = (user: UserDTO) => {
           </a>
         )}
       </td>
+      {showLicensedRole && (
+        <td className={cx('link-td', styles.iconRow)}>
+          <a className="ellipsis" href={editUrl} title={user.name} aria-label={`Edit user's ${user.name} details`}>
```

Comment:
Hmm, thinking about all these repetitive aria-labels, can we find a way to reduce them?

Reply:
I guess they'd be abstracted into a function.

Reply:
That could be a good start 👍